tests/unittests/main.c

```diff
@@ -21,6 +22,10 @@
 
 int main(void)
 {
+#ifdef MODULE_XTIMER
+    xtimer_init();
```

Comment:
Please add a comment in code why we run this here (auto-init is disabled in unittests, and modules depending on xtimer expect it to be initialized)